packages/components/app/styles/components/link/standalone.scss

```diff
@@ -21,6 +21,8 @@ $hds-link-standalone-border-width: 1px;
   align-items: center;
   justify-content: center;
   width: fit-content;
+  padding-top: 4px;
+  padding-bottom: 4px;
```

Comment:
Can convert these to tokens, if helpful 🤷

Reply:
Nah probably not for now. _Hopefully_ (TM) we won't be going back and visiting this in the near future.